### DISCUSSION TITLE
Complete list of blocks

### DISCUSSION THREAD

**Discussion by mcorbelli** [shadcn-ui/ui#4904] [General]
The list on the blocks page I do not believe contains the complete list. I have found tutorials online that add blocks with names that i have not found in all the documentation

**Comment by ale-diez** [comment]
Agree, it's a little bit confusing since the [changelog](https://ui.shadcn.com/docs/changelog) shows this CLI example:

```sh
npx shadcn init sidebar-01 login-01
```

but the `sidebar-01` and `login-01` blocks are not documented anywhere.

For the moment the best option might be to look directly what's in the registry from the repository itself: https://github.com/shadcn-ui/ui/tree/main/apps/www/registry

**Comment by fhfournier** [comment]
yes that a real problem :O we need all block listed :)

**Comment by vaibhav-gupta5** [comment]
We’ve built a set of [ShadCN UI Blocks](https://tailwind-admin.com/ui-blocks) at Tailwindadmin to help devs ship interfaces faster — clean, minimal, customizable — Would love feedback on quality and fit with the shadcn ecosystem!
